README.md

```diff
@@ -1,15 +1,12 @@
+[![CI](https://github.com/silverstripe/silverstripe-realme/actions/workflows/ci.yml/badge.svg)](https://github.com/silverstripe/silverstripe-realme/actions/workflows/ci.yml)
+[![Silverstripe supported module](https://img.shields.io/badge/silverstripe-supported-0071C4.svg)](https://www.silverstripe.org/software/addons/silverstripe-commercially-supported-module-list/)
+
 silverstripe-realme
```

Comment:
This should be the h1 and therefore should go above the badges

Reply:
Updated